src/lib/image/bootstrap/busybox/busybox.c

```diff
@@ -0,0 +1,45 @@
+/* 
```

Comment:
ok, I'll take care of it. I also may reorganize some of the bootstrap code (I don't like it under src/lib/image as it breaks the nomenclature <-> directory standard I've been using). For example: singularity_bootstrap_yum() should be in lib/bootstrap/yum. I will take care of that too but if you find yourself with extra time, and bored, and this hasn't been committed yet, ping me in Slack. It is likely I'll get interrupted. lol